src/main/java/com/alkemy/ong/application/rest/request/NewsRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.alkemy.ong.application.rest.request;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class NewsRequest {
+
+  @Size(max = 50)
+  @Pattern(regexp = "[a-zA-Z\\s]+", message = "name must contain letters and blank spaces")
+  @NotEmpty(message = "name may not be empty")
+  private String name;
+
+  @Pattern(regexp = "[a-zA-Z0-9\\s]+", message = "text must contain letters and numbers")
+  @NotEmpty(message = "text may not be empty")
+  private String text;
+
+  @Pattern(regexp = "[0-9]+", message = "image must contain numbers")
```

Comment:
Este pattern no condice con lo solicitado en el ticket. Debería ser: ^[a-zA-Z0-9_]*$

El mensaje debería ser: "The image has invalid format." o "The image accepts only alphanumeric characters."